service/subprocessmanager/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/open-telemetry/opentelemetry-collector/service/subprocessmanager
+
+go 1.15
```

Comment:
Why not 1.14?

Reply:
In core we have one module, so no need to have a submodule for this

Reply:
`1.15` was generated with `go mod init`, but okay didn't know it was all in one module! thanks 👍